src/System.Security.Cryptography.Primitives/tests/HashAlgorithmNameTests.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Security.Cryptography.Primitives.Tests
+{
+    public static class HashAlgorithmNameTests
+    {
+        [Theory]
+        [MemberData(nameof(ValidInputs))]
+        public static void FromOid_ValidInput(string oid, HashAlgorithmName expected)
+        {
+            Assert.Equal(expected, HashAlgorithmName.FromOid(oid));
+        }
+
+        [Fact]
+        public static void TryFromOid_ThrowsForNullInput()
```

Comment:
The name of this one, and the next one, say Try, but aren't for the Try.

Reply:
Oops, fixed.